fastify.js

```diff
@@ -93,7 +131,7 @@ function build () {
       const handle = node[req.method]
 
       if (handle && req.method === 'GET') {
-        handleNode(handle, params, req, res, null)
+        handleNode(handle, params, req, res, urlUtil.parse(req.url, true).query)
```

Comment:
I know, but since this pr is not trivial, for the moment I'm using the same variable to get it work.
In your opinion which is the best way to handle this?